examples/uo_teaser_project/geojson_8_buildings.json

```diff
@@ -210,7 +210,7 @@
         "id": "6",
         "name": "Restaurant 15",
         "type": "Building",
-        "building_type": "Food service",
+        "building_type": "Office",
```

Comment:
We have to wait until a new release of TEASER to fix an error with a NoneType central AHU system.